pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: webdriver
-version: 2.0.1-dev
+version: 2.1.0
 authors:
   - Marc Fisher II <fisherii@google.com>
   - Matt Staats<staats@google.com>
-description: >-
-  Provides WebDriver bindings for Dart. These support the WebDriver JSON
-  interface and the W3C spec and require the use of the WebDriver remote server.
+description: >
+  Provides WebDriver bindings for Dart. These support the WebDriver JSON interface
+  and the W3C spec and require the use of the WebDriver remote server.
```

Comment:
"Supports WebDriver JSON interface and W3C spec. Requires the use of WebDriver remote server."